utils/rspamd_stats.pl

```diff
@@ -47,6 +48,7 @@
     "ignore=s@"             => \@symbols_ignored,
     "group|g=s@"            => \@symbols_groups,
     "log|l=s"               => \$log_file,
+    "mult=s@"               => \%symbols_mult,
```

Comment:
Not sure if s@ works for % hash tables, but I might be wrong...

Reply:
According to https://perldoc.perl.org/Getopt::Long#Options-with-multiple-values neither `@` nor `%` are needed if passing direct references to arrays/hashes.